lib/generator/input_object.dart

```diff
@@ -2,55 +2,84 @@ import 'package:built_collection/built_collection.dart';
 import 'package:code_builder/code_builder.dart';
 import 'package:graphql_generator/generator/helper.dart';
 import 'package:graphql_generator/generator/model.dart';
+import 'package:analyzer/dart/element/type.dart';
 
 class InputObjectGenerator {
 
-  String namespace = '';
-  List<ObjectType> enumTypes = [];
+  InputObjectGenerator() {}
 
-  InputObjectGenerator() {
-  }
-
-  inputObjectGenerator(List<ObjectType> inputObjectTypes, String namespace,
-      List<ObjectType> enumTypes) {
-    this.namespace = namespace;
-    this.enumTypes = enumTypes;
+  generate(
+      Map<String, DartType> types,
+      List<ObjectType> enumTypes,
+      List<ObjectType> interfaceTypes,
+      List<ObjectType> unionTypes,
+      List<ObjectType> objectTypes,
+      List<ObjectType> inputObjectTypes,
+      {String namespace = ""}) {
     Map<String, Class> classes = {};
     inputObjectTypes.forEach((typeObject) {
-      classes.putIfAbsent(
-          '$namespace${typeObject.name}', () {
-        return _generateClass(typeObject);
+      classes.putIfAbsent('$namespace${typeObject.name}', () {
```

Comment:
Seems to me like you can replace this with an `inputObjectTypes.map((objType) => _generateClass(...)`. Why do we have `putIfPresent` if the types are guaranteed to be unique? It's an unnecessary check. Also see previous comment on simplifying the return type to `Iterable<Class>`